hopp/tools/dispatch/csp_pv_battery_plot.py

```diff
@@ -84,7 +83,8 @@ def init_hybrid_plant():
     site = SiteInfo(site_data,
```

Comment:
This may not be the time to do it, but I would prefer we move to the convention that function calls that need to be broken get moved to just 1 indent, like below. Also, to add the trailing comma on the last input. But this could be handled with commit hooks.

```
site = SiteInfo(
    site_data,
    solar_resource_file=solar_file,
    grid_resource_file=prices_file,
    desired_schedule=desired_schedule
    desired_schedule=desired_schedule,
    wind=False,
)
```

Reply:
Will be fixed with autoformatting PR